app/model/profile.model.js

```diff
@@ -279,16 +269,10 @@ function loadCertificateSubEntities(headers) {
     return function(certificates) {
         return new Promise(function(resolve) {
             var skills = skillResource.getAllSkills(headers);
-            var customers = customerResource.getAllCustomers(headers);
-            var domains = domainResource.getAllDomains(headers);
-            return Promise.all([skills, customers, domains])
+            return skills
```

Comment:
no need to assign skills like this. Use skillResourse.getAllSkills(headers).then(function(skills){} instead.
This way you wont need to use "skills.value()" and such. The only time we assign to variables are when we use several promises; i/e promise.all().